eth/eventhandler/validation.go

```diff
@@ -0,0 +1,76 @@
+package eventhandler
+
+import (
+	"fmt"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/herumi/bls-eth-go-binary/bls"
+
+	registrystorage "github.com/bloxapp/ssv/registry/storage"
+	"github.com/bloxapp/ssv/storage/basedb"
+)
+
+const maxOperators = 13
+
+func (eh *EventHandler) validateOperators(txn basedb.Txn, operators []uint64) error {
+	operatorCount := len(operators)
+
+	if operatorCount > maxOperators {
+		return fmt.Errorf("too many operators (%d)", operatorCount)
+	}
+
+	if operatorCount == 0 {
+		return fmt.Errorf("no operators")
+	}
+
+	canBeQuorum := func(v int) bool {
+		return (v-1)%3 == 0 && (v-1)/3 != 0
+	}
+
+	if !canBeQuorum(len(operators)) {
+		return fmt.Errorf("given operator count (%d) cannot build a 3f+1 quorum", operatorCount)
+	}
```

Comment:
Can we please move this func under [this one](https://github.com/bloxapp/ssv/blob/90874aefde6f266d476f6c74ff3ed066ed136b58/protocol/v2/types/ssvshare.go#L82-L85) (maybe call it `ValidCommitteeSize`) and add a test for it there?

Reason is because it's related to that func so it's better together with it (so if we make modification to any of them we can't miss the other).

Reply:
done